src/components/dataset/introduction.vue

```diff
@@ -0,0 +1,79 @@
+<!--
+Copyright 2022 ODK Central Developers
+See the NOTICE file at the top-level directory of this distribution and at
+https://github.com/getodk/central-frontend/blob/master/NOTICE.
+
+This file is part of ODK Central. It is subject to the license terms in
+the LICENSE file found in the top-level directory of this distribution and at
+https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+including this file, may be copied, modified, propagated, or distributed
+except according to the terms contained in the LICENSE file.
+-->
+<template>
+  <div id="dataset-introduction" class="panel-dialog">
+    <div class="panel-heading">
+      <span class="panel-title">{{ $t('title') }}</span>
+    </div>
+    <div class="panel-body">
+      <p>{{ $t('body[0]') }}</p>
+      <i18n-t tag="p" keypath="body[1]">
+        <template #icon1><span class="step-number">(1)</span></template>
```

Comment:
The number and parentheses are intentionally not internationalized. See the [discussion](https://docs.google.com/document/d/1naxyzFU8EjcWKfYzE_mEF5cY38l1QRNUZ_0VW2Gp-r0/edit?disco=AAAAjSbSdz0) in the release criteria.